src/server/generic_family_test.cc

```diff
@@ -354,4 +364,16 @@ TEST_F(GenericFamilyTest, Time) {
   }
 }
 
+TEST_F(GenericFamilyTest, Persis) {
```

Comment:
Persist

Reply:
will change - strange that didn't see it in the spell checker